contracts/zero-ex/contracts/src/errors/LibCommonRichErrors.sol

```diff
@@ -0,0 +1,36 @@
+/*
+
+  Copyright 2020 ZeroEx Intl.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
+pragma solidity ^0.6.5;
+
+
+library LibCommonRichErrors {
+
+    // solhint-disable func-name-mixedcase
+
+    function OnlyCallableBySelfError(address sender)
+        internal
+        pure
+        returns (bytes memory)
+    {
+        return abi.encodeWithSelector(
+            bytes4(keccak256("OnlyCallableBySelfError(address)")),
```

Comment:
ya, I started inlining this computation:
- It only saves <100 gas to precompute the selector
- It was so annoying to precompute the selector
- This is less error prone
- This code rarely executes anyway